src/Emulator/Tabs/ConfigureTab/ConfigureTab.js

```diff
@@ -0,0 +1,136 @@
+import React, { useState, useRef } from 'react';
+import './ConfigureTab.css';
+import DeviceContainer from './DeviceContainer/DeviceContainer';
+
+const MAX_DEVICES = 10;
+
+// Sample responses from currently non-existent API
+// These represent the starting config for a lesson
+const routerConfig = [
+  {
+    name: 'r1',
+    connections: ['s1', 's2'],
+  },
+];
+const switchConfig = [
+  {
+    name: 's1',
+    connections: ['r1', 'h1', 'h2', 'h3'],
+  },
+  {
+    name: 's2',
+    connections: [],
+  },
+];
+const hostConfig = [
+  {
+    name: 'h1',
+    connections: ['s1'],
+  },
+  {
+    name: 'h2',
+    connections: ['s1'],
+  },
+  {
+    name: 'h3',
+    connections: ['s1'],
+  },
+  {
+    name: 'h4',
+    connections: [],
+  },
+];
+
+const getNextNumber = (s) => Number(s.slice(1)) + 1;
+const getNextDeviceName = (device) => {
+  const lastDeviceName = device[device.length - 1].name;
```

Comment:
@sammytran277 I misspoke the instructions because I remembered my scenario wrong, this is the piece that is worrying me - this would be `devices[-1]` if we started with a config of no items and added `1` if javascript works the way I think it does. Please check it out! This can be a unit test potentially.

Reply:
Fixed in a new commit, unit tests will be coming shortly.